### PR TITLE
[4.1] Workflow strings code style

### PR DESCRIPTION
**Pull request by brianteeman**
Updates a few strings in accordance with the language style guide. Sentences should end with a period and labels should be capitalised.

Code review

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 41003175c7829810680761fa3be6d17434fbb4e8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37741">issues.joomla.org/tracker/joomla-cms/37741</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 41003175c7829810680761fa3be6d17434fbb4e8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37741">issues.joomla.org/tracker/joomla-cms/37741</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37741">issues.joomla.org/tracker/joomla-cms/37741</a>.</sub>

**Comment by bembelimen** [comment]
Thx

File: administrator/language/en-GB/plg_workflow_notification.ini [hunk 1]
```diff
@@ -7,14 +7,14 @@ COM_WORKFLOW_BASIC_STAGE="Basic Stage"
 COM_WORKFLOW_NOTIFICATION_FIELDSET_LABEL="Notification"
 PLG_WORKFLOW_NOTIFICATION="Workflow - Notification"
 PLG_WORKFLOW_NOTIFICATION_ADDTEXT="The stage has changed"
-PLG_WORKFLOW_NOTIFICATION_ADDTEXT_DESC="This text will be sent: Title [title], changed by [user], new state: [state]. You can add own text to this information. You can localise the text by using a language key and creating a language override."
+PLG_WORKFLOW_NOTIFICATION_ADDTEXT_DESC="This text will be sent: Title [title], changed by [user], new state: [state]. You can add your own text to this message and you can localise the text by using a language key and creating a language override."
```
Comment (Kostelano) [inline administrator/language/en-GB/plg_workflow_notification.ini:10]: This is the old wording, PR https://github.com/joomla/joomla-cms/pull/35775 extended the message text by one more parameter - transition. In the future, a [transition] could also be added.

**Pull request merged**